src/main/java/org/jitsi/jicofo/jigasi/JigasiDetector.java

```diff
@@ -50,7 +50,7 @@
     /**
      * The local region of the this jicofo instance.
      */
-    private final String localRegion;
+    private final String localRegion = JicofoConfig.config.localRegion();
```

Comment:
Do you think we should get rid of the member and use `JicofoConfig.config.localRegion()` in each place?

Reply:
I don't know, I have to think about it more. My first thought is that, unless the config can change at runtime, I prefer to initialize it once. Being able to change things at runtime would be nice, but it would require a larger effort. I'd leave this change for the time when we decide to do that.

Reply:
Sure...but that's already implemented in the metaconfig layer (we only try retrieving a value once and then cache the result) if that makes you feel better.